crates/search/src/chunking.rs

```diff
@@ -361,13 +416,13 @@ impl SearchIndex for ChunkedSearchIndex {
 }
 
 fn to_offset_array(x: &[Vec<(usize, usize)>], nullable: bool) -> FixedSizeListArray {
-    let mut builder = FixedSizeListBuilder::new(UInt64Builder::new(), 2)
```

Comment:
This is just a bug, it did't affect the writing to S3, but now that we are reading them, we get schema mismatches.